polars/polars-core/src/frame/mod.rs

```diff
@@ -394,6 +394,22 @@ impl DataFrame {
     }
 
     /// Get the data types of the columns in the DataFrame.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use polars_core::df;         // or "use polars::df"
+    /// use polars_core::prelude::*; // or "use polars::prelude::*"
+    ///
+    /// fn example() -> Result<()> {
+    ///     let venus_air: DataFrame = df!("Element" => &["Carbon dioxide", "Nitrogen"],
+    ///                                    "Fraction" => &[0.965, 0.035])?;
+    ///     
+    ///     assert_eq!(venus_air.dtypes(), vec![DataType::Utf8, DataType::Float64]);
```

Comment:
Nit: maybe we could compare with a slice `&[DataType..]. That saves an allocation.